lib/optimizely/project_config.rb

```diff
@@ -291,7 +291,7 @@ def get_forced_variation(experiment_key, user_id)
       experiment_id = experiment['id'] if experiment
       # check for nil and empty string experiment ID
       # this case is logged in get_experiment_from_key
-      return nil if experiment_id.nil? || experiment_id.empty?
+      return nil if !experiment_id.is_a?(String) || experiment_id.empty?
```

Comment:
Please confirm that this change is really needed because from the code I can see that experiment_id will either be nil or a valid string from the data file.